src/main/features/claim/views/fees.njk

```diff
@@ -55,9 +55,9 @@
               </thead>
               {% for row in rows %}
                 <tr>
-                  <td>{{ row.from | pennies2pounds | numeral }} to {{ row.to | pennies2pounds | numeral }}</td>
-                  <td>{{ row.fees[1] | pennies2pounds | numeral }}</td>
-                  <td>{{ row.fees[2] | pennies2pounds | numeral }}</td>
+                  <td>{{ row.minRange }} to {{ row.maxRange }}</td>
+                  <td>{{ row.fees[1] | default('-') }}</td>
```

Comment:
As we talked it makes sense to as UX what would they do, whether `-` is the best we can do.

Reply:
Spoken to @tarik-johnston1 - He has suggested putting 'Not available' instead of '-' for this edge case. All actioned! Thanks